presets/jscs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "preset": "google",
```

Comment:
Oh, gotcha. Super confusing. This isn't the name of the preset itself, this is a case of presets extending each other.